data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+		payment 180000
+
+event "coalition citizens discover fangvin"
+
+mission "Fangvin: Book Discovered"
+	name "Look for Anna Fangvin"
+	description "Get in contact with Anna Fangvin, an author in which the group of Coalition citizens have shown interest in."
+	source
+		government "Coalition"
+	destination "Zug"
+	to offer
+		has "event: coalition citizens discover fangvin"
+	on offer
+		conversation
+			`Upon landing in the spaceport, you get a message from the same group of Coalition citizens. You recall that you had provided them with a datachip containing the entire knowledge of humanity.`
+			``
+			`		Dear <first>,`
+			`	Interested in an author from the datachip, we are. 'Anna Fangvin', her name is. The story about your race's actions against the oppressors, captivate us it did. To converse with her, we wish.`
+			choice
+				`	(Disregard.)`
+					decline
+				`	(Look for Anna Fangvin.)`
+			`A quick look through your encyclopedia reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview and set a course.`
+	on complete
+		conversation
+			`As you prepare to land, you write Anna a short message, offering an interview. In less than 10 minutes you receive a reply, explaining that Anna has accepted your offer and will wait for you in the spaceport.`
+			`	Look for Anna in the spaceport, so you can take her to the Coalition for a little conversation with those Coalition citizens who showed interest in her work.`
+
+mission "Fangvin: Struggling Author 1"
+	name "Take Anna to Coalition Space"
+	description "Take Anna Fangvin to <destination>, where she might find success as an author after having struggled on <origin> for so long."
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		has "Fangvin: Book Discovered: done"
+	on offer
+		conversation
+			`Upon entering the spaceport, you walk past a restaurant. A woman then raises her arms, waving towards you. She must be Anna Fangvin, that author the Coalition citizens showed interest in.`
+			choice
+				`	(Approach her.)`
+			`	You approach the woman and sit right next to her. "Hello, are you <first>?`
+			choice
+				`	"Yes, I am. So, what do you want to ask?"`
+			choice
+				`	"What do you write?"`
+					goto "what do you write"
+			label "what do you write"
+			`	"I write science fiction novels. My latest novel was 'Beyond the Hyperdrive', which I released a few weeks ago. It's about a bunch of humans who steal jump drives from the Quarg. Here, have a copy."`
+			`	Anna retrieves a hardcover book from her bag. It has an image of a Jump Drive on it. You take the book.`
+			`	"You'll like it, I promise."`
+			choice
+				`	"How long is the book?`
+					goto "how long is the book"
+				`	"Why is it a story about humanity stealing from the Quarg?"`
+					goto "anna hates quarg"
+			label "how long is the book"
+			`	It isn't very long, it should be three hundred pages.`
+			label "anna hates quarg"
```

Comment:
Also, the "how long is the book" label follows immediately with the "anna hates quarg" label, but there's nothing connecting there, so need to either get some alternative to reaching that choice or just re-offering the choice with only the Quarg option.